### DISCUSSION TITLE
Renaming an Instance issues

### DISCUSSION THREAD

**Discussion by ibanks42** [pockethost/pockethost#239] [Q&A] [answered]
In the Danger Zone the link for more information for renaming an instance points to `https://pockethost.gitbook.com/manual/usage/rename-instnace.md`
The correct link is `https://pockethost.gitbook.io/manual/daily-usage/rename-instance`.

Also, renaming an instance seems to not do anything currently.

**Comment by benallfree** [comment]
@isaiah-banks Thanks for the report. Feel welcome to send a PR for that first issue.

I'll need to look into the instance renaming.

**Reply by ibanks42** [reply]
Sent PR #240

**Reply by ibanks42** [reply]
After trying to rename an instance again I've realized renaming works as long as there's no numbers in the name.

**Reply by benallfree** [reply, marked as answer]
Ah, yes I'm sorry. This ux has been made clearer in release v0.8.1. See #242

**Reply by ibanks42** [reply]
Yay errors :)

I assume there's a reason to not allow numbers in the name of instances? Just curious. Is it security related?

**Reply by ibanks42** [reply]
Ah never mind I see, it's part of the roadmap for #25 and #42

**Reply by benallfree** [reply]
> not allow numbers in the name of instances

I can't remember why I originally made this decision, but it seems like something that could be changed. Want to take a shot at a PR?

* Come up with a new Regex
* Modify frontend and RPC validation to test for regex compatibility
* When deployed, I'll add the regex to the PocketBase collection rules as a third measure of security.
